app/controllers/plantings_controller.rb

```diff
@@ -31,7 +31,7 @@ def show
     @planting = Planting.includes(:owner, :crop, :garden, :photos)
       .friendly
       .find(params[:id])
-    @photos = @planting.photos.order(created_at: :desc).includes(:owner).paginate(page: params[:page])
+    @photos = @planting.photos.order(date_taken: :asc).includes(:owner).paginate(page: params[:page])
```

Comment:
Can we keep it as descending order? Newest photos first.

Reply:
for sure! 
changing now.